tests/test_pagination.py

```diff
@@ -648,6 +648,16 @@ def test_cursor_pagination(self):
 
         assert isinstance(self.pagination.to_html(), type(''))
 
+    def test_offset_cutoff(self):
+        (previous, current, next, previous_url, next_url) = self.get_pages('/')
+
+        next_url = self.pagination.encode_cursor(Cursor(1050, False, None))
+        (previous, current, next, previous_url, next_url) = self.get_pages(next_url)
+
+        assert previous == [7, 7, 7, 8, 9]
```

Comment:
I don't think what this is correct, since current page is outside of range and is empty, while previous page contains items from penultimate page, not the last.

But at least, this doesn't lead to exception.